### PR TITLE
Env rotation

### LINKED ISSUE #68: Add option of rotating the lighting environment
This is useful when testing materials under various lighting conditions.

### PR DESCRIPTION
**Pull request by slimbuck**
Fixes #68

This PR includes the following changes:
- add a slider for rotating the lighting environment
- disable alpha on render target to address incorrectly rendered glass
- update to pcui v1.0.8
- update 'local' npm script to pull in local engine
- add 'local:dbg' to pull in debug engine

I confirm I have signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,6 +80,7 @@
     "build:watch": "webpack --config webpack.config.js --watch",
     "build:server": "ENVIRONMENT=production PUBLIC_PATH=/viewer/static/ webpack --config webpack.config.js",
     "build:analyze": "ENVIRONMENT=production ANALYZE_BUNDLE=true webpack --config webpack.config.js",
-    "local": "concurrently --kill-others \"npm run build:watch\" \"npm run serve\""
+    "local": "ENGINE_PATH=../engine/build/playcanvas.js EXTRAS_PATH=../engine/build/playcanvas-extras.js concurrently --kill-others \"npm run build:watch\" \"npm run serve\"",
```
Comment (willeastcott) [inline package.json:83]: Specifying env vars like this won't work on Windows, right?

Comment (slimbuck) [reply package.json:83]: I haven't a clue! :D

Comment (willeastcott) [reply package.json:83]: Let me rephrase. This won't work on Windows. 🤣

Comment (slimbuck) [reply package.json:83]: Any idea if this will fail or will silently be ignored on windows? If windows silently ignores then I'd say it's probably ok.

Looks like windows users must do `set ENGINE_PATH=blah' before running the command to get the same result.

Comment (slimbuck) [reply package.json:83]: Just found this, will try it https://www.npmjs.com/package/cross-env

Comment (slimbuck) [reply package.json]: That seems to work on mac and according to the docs should work elsewhere too. (Though this could do with testing on windows).

**Review by ellthompson — APPROVED** [review]

**Review by mvaligursky — APPROVED** [review]
